feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/services/_index.md

---
title: Services
weight: 7
description: >-

>   You will find here information of how to configure and use services in Beagle Flutter.

Reviewer: ```suggestion
In this section, you will find information on how to configure and use services in Beagle Flutter.
```
